src/main/java/com/bitso/Bitso.java

```diff
@@ -208,12 +217,32 @@ public BitsoFunding getUserFundings(String... fundingsIds){
         String json = sendBitsoGet(request);
         JSONObject o = Helpers.parseJson(json);
         if(o == null || o.has("error")){
-            logError("Error getting user withdrawals");
+            logError("Error getting user fundings");
             return null;
         }
         return new BitsoFunding(o);
     }
 
+    public BitsoTrade getUserTrades(String... tradesIds){
+        String request = "/api/v3/user_trades";
+        if(tradesIds.length > 0){
+            request += "/";
+            for (String string : tradesIds) {
+                request += string + "-";
+            }
+            request = request.substring(0, request.length() - 1);
```

Comment:
previous for leaves request string with a "-" at the end, substring funtion removes that character in order to get the correct string.